pkg/volume/volume.go

```diff
@@ -170,7 +170,7 @@ type CustomBlockVolumeMapper interface {
 	// For most in-tree plugins, attacher.Attach() and attacher.WaitForAttach()
 	// will do necessary works.
 	// This may be called more than once, so implementations must be idempotent.
-	SetUpDevice() error
+	SetUpDevice() (string, error)
```

Comment:
Add a comment explaining what the return values represent

Reply:
fixed.